### ISSUE TITLE
Error in get("GO2ONT", envir = GO_DATA) : object 'GO2ONT' not found

### ISSUE THREAD

**Issue by jarpin90** [YuLab-SMU/clusterProfiler#498]
Thanks professor. 

I have been using clusterProfiler for 3 months, and I have never experienced this error before. 
It only happens when aiming at doing a GSEA from an ENSEMBL gene list. My list is ordered and preset and works well with other functions in the package. I am using a pvalueCutoff = 0.01, and 1000 permutations. 

Could you please help me out?  
I need this with urgency for a presentation next week, and because of a system crash (while running the package, I've lost my progress in the working environment)

## here is my code: 
gse.CADi <- gseGO(geneList = gene_list_CADi,
                 ont = "ALL",
                 keyType = "ENSEMBL", 
                 minGSSize = 50, 
                 maxGSSize = 800, 
                 pAdjustMethod = "BH",
                 pvalueCutoff = 0.01,
                 verbose = TRUE, 
                 nPermSimple=10000,
                 OrgDb = organism)

Thank you again and all the best,

**Comment by huerqiang** [comment]
Please provide repeatable code with the data.

**Comment by jarpin90** [comment]
Hi Erquiang, 
This is the code: 
gseGO(geneList = "data"
                 ont = "ALL",
                 keyType = "ENSEMBL", 
                 minGSSize = 50, 
                 maxGSSize = 800, 
                 pAdjustMethod = "BH",
                 pvalueCutoff = 0.05,
                 verbose = TRUE, 
                 nPermSimple=1000,
                 OrgDb = organism)
"data"= geneList, ordered in descending logFC with Enseml IDs,
all the rest are the settings included in the gseGO function. 
Thanks

**Comment by huerqiang** [comment]
The code works well with me, did you use the latest version of `clusterProfiler`?
![image](https://user-images.githubusercontent.com/48857018/187119234-6e9a4ebd-3183-411c-bb05-2cb50e5cec4c.png)

**Comment by jarpin90** [comment]
Yes, I am using the version 4.4.4. Should I be using an older version instead? Thank you 
`> packageVersion('clusterProfiler')
[1] 4.4.4`

**Comment by huerqiang** [comment]
Please try the Github version. 
If you still have this error, please send the data and the complete code to my email: 13766876214@163.com.

**Comment by milomilomic** [comment]
Hi all. 

Has this issue been solved yet? I keep getting this exact error. Please note that I am relatively new to R/R Studio and this is my first time trying out clusterProfiler, so apologies in advance. 

I used a dataframe containing UNIPROT in the first column and logFC in the second, prepared a genelist and sorted it. I'm  unsure on how I should preferably provide sample data to reproduce the error, but I also got this same error whilst working with drosophila sample date from "https://learn.gencore.bio.nyu.edu/wp-content/uploads/2019/10/drosphila_example_de.csv" (and adjusted the organism accordingly). 
I really hope someone can help. Thanks in advance. 

My code is as follows:

organism = "org.Hs.eg.db"
BiocManager::install("org.Hs.eg.db", character.only = TRUE)
library(organism, character.only=TRUE)

gse <- gseGO(geneList=gene_list, 
             ont ="ALL", 
             OrgDb= organism,
             keyType = "UNIPROT",
             minGSSize = 3, 
             maxGSSize = 800, 
             pvalueCutoff = 0.05, 
             verbose = TRUE, 
             pAdjustMethod = "none")

**Comment by milomilomic** [comment]
Hi all, 
For me the issue was solved once i switched ont="ALL" to only one component, for instance ont="BP" 
Hope that's helpful for anyone

**Comment by zddda58** [comment]
> 大家好，对我来说，一旦我将 ont=“ALL” 切换到一个组件，问题就解决了，例如 ont=“BP” 希望这对任何人都有帮助

Hello, I got the same error. I don't know, but your approach really worked. Thank you so much!!!

**Comment by GuangchuangYu** [comment]
Duplicate of #467 and verified fixed the same way: `enrichGO(ont = "ALL")` on current devel no longer raises `object 'GO2ONT' not found` (commit aca51c9, "use enrichit"). Closing.
